.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:latest
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: foreign_travel_content_checker_test
+        ports:
+          - 5432:5432
+        options:
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup Ruby and install gems
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+
+      - name: Setup the database
+        env:
+          RAILS_ENV: test
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: foreign_travel_content_checker_test
+        run: |
+          bundle exec rails db:migrate
+          bundle exec rails db:seed
```

Comment:
You'll probably want this to create the database too. You can do `db:reset` which will drop, create and seed.

It's not recommended to using db:migrate as a way to set-up a database. I've not found the authoritative documentation of this but this one at least sets the tone: https://edgeguides.rubyonrails.org/active_record_migrations.html#old-migrations

Reply:
Thanks - fixed